backend_new/src/dtos/listings/listing-multiselect-question.dto.ts

```diff
@@ -0,0 +1,20 @@
+import { MultiselectQuestion } from '../multiselect-questions/multiselect-question.dto';
+import { Expose, Type } from 'class-transformer';
+import { IsNumber, IsDefined } from 'class-validator';
+import { ValidationsGroupsEnum } from '../../enums/shared/validation-groups-enum';
+import { ListingGet } from './listing-get.dto';
+
+export class ListingMultiselectQuestion {
+  @Type(() => ListingGet)
+  @IsDefined({ groups: [ValidationsGroupsEnum.default] })
+  listings: ListingGet;
+
+  @Expose()
+  @Type(() => MultiselectQuestion)
+  @IsDefined({ groups: [ValidationsGroupsEnum.default] })
+  multiselectQuestions: MultiselectQuestion;
```

Comment:
curious why this is named plural but typed singular

Reply:
if you peep the `schema.prisma` for `model ListingMultiselectQuestions` in order to duck having to manual convert the `.findmany()`/`.findFirst()` I just kept the name the same as what prisma is going to give us 

I think with the standarization ticketo this kinda stuff will go away and get standarized